rfdetr/detr.py

```diff
@@ -39,11 +40,32 @@ def export(self, **kwargs):
         self.model.export(**kwargs)
 
     def train_from_config(self, config: TrainConfig, **kwargs):
-        with open(
-            os.path.join(config.dataset_dir, "train", "_annotations.coco.json"), "r"
-        ) as f:
-            anns = json.load(f)
-            num_classes = len(anns["categories"])
+        # Check if dataset is in YOLO format
```

Comment:
let's drop redundant python code comments; in general let's try to make it self descriptive